templates/arbiter/01-arbiter-kubelet/_base/units/kubelet.service.yaml

```diff
@@ -10,7 +10,7 @@ contents: |
   [Service]
   Type=notify
   ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
-  ExecStartPre=-/usr/sbin/restorecon /usr/local/bin/kubenswrapper /usr/bin/kubensenter
+  ExecStartPre=-/usr/sbin/restorecon -riv /var/lib/kubelet/pod-resources /usr/local/bin/kubenswrapper /usr/bin/kubensenter
```

Comment:
We should take out the `v`. The 'v' adds additional logging, and Nodes are already under logging pressure. Let's take out the v since there could be an abundance of files that are changed.

Reply:
>  Let's take out the v since there could be an abundance of files that are changed.

That won't happen anymore because there is only one file (socket) under the `/var/lib/kubelet/pod-resources` path :)

But I'm OK with removing it anyway